packages/ibm-products/src/components/EditInPlace/EditInPlace.tsx

```diff
@@ -129,6 +129,10 @@ interface EditInplaceProps extends PropsWithChildren {
    * current value of the input
    */
   value: string;
+  /**
+   * placeholder for the input
+   */
+  placeholder: string;
```

Comment:
```suggestion
  placeholder?: string;
```
this is optional

Reply:
Thanks @devadula-nandan.
The changes are updated.